.prettierrc

```
@@ -0,0 +1,11 @@
+{
+	"arrowParens": "always",
+	"bracketSpacing": true,
+	"experimentalTernaries": true,
+	"jsxSingleQuote": true,
```

Personal preference of mine.
I prefer my HTML with double quotes.

```suggestion
	"jsxSingleQuote": false,
```